### PR TITLE
Release v4.5.0-rc1

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

**Review by daira — APPROVED** [review]
utACK

I would have preferred the release PR to be separate from the typecode change, but I don't need them to be separated now that it's been done this way.

**Comment by therealyingtong** [comment]
utACK.

**Review by therealyingtong — APPROVED** [review]
utACK

File: doc/release-notes/release-notes-4.5.0-rc1.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+      Update depends/packages/native_rust.mk
+
+Marshall Gaucher (5):
+      Remove sprout funding flow logic
+      Add fix and note for timing issue
+      Update funding logic bug
+      Add usage documentation for manual and faucet driven tests
+      Update funding logic
+
+Jack Grigg (12):
```
[the remark this answers is not in the document]
Comment (str4d) [reply doc/release-notes/release-notes-4.5.0-rc1.md:298]: Probably because it isn't de-duplicating emails correctly.

**Pull request merged**